nixos/modules/services/backup/btrbk.nix

```diff
@@ -1,18 +1,21 @@
 { config, pkgs, lib, ... }:
+
+with lib;
```

Comment:
There are arguments against this in https://github.com/NixOS/nixpkgs/pull/101139

Reply:
Fixed.